lib/smart_answer/date_range.rb

```diff
@@ -102,5 +108,13 @@ def leap_dates
     def leap?
       leap_dates.any?
     end
+
+    def weeks_later(weeks)
```

Comment:
`date_range.increase_weeks 10` might make more sense?

Reply:
Having rethought this I recommend we rename this to `.weeks_after` because `later` always suggests in the future. But we are using it with minus X weeks to go backwards in a date range, which doesn't fit right with `later`. 

I think`.weeks_after(-10)` feels a bit more flexible and appropriate.

Reply:
@chao-xian what about `.shift_weeks(-10)` ?

Reply:
This is just returning a result though, not setting a new value and that's what shift means to me. Ideally it would read as something like `Date range at +/- X weeks`

Mabe just swap the words in my suggestion so it'd be `date_range.after_weeks(10)` and equally `date_range.after_weeks(-10)` I think would work. What do you think of that?

Reply:
@chao-xian that is fine 👍

Reply:
@chao-xian but I would prefer `weeks_after`...

Reply:
Sorry @mgrassotti I didn't realise that you'd addressed my comments about renaming the methods in [this commit](2e3fd972701ca9637903539064e3c6890f699f4d). 

I will approve this PR now but I think it would have been better to make that amendment in the commit where you introduced the new code.

Thanks for your work!